unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs

```diff
@@ -56,6 +59,11 @@ public class ExploreV2MenuComponentController : IExploreV2MenuComponentControlle
 
     public void Initialize()
     {
+        #if UNITY_EDITOR
+        // Prevent warning when starting on unity editor mode
+        // TODO: Are we instantiating 500 different kinds of animations?
+        DOTween.SetTweensCapacity(500,50);
```

Comment:
Please move it to `Main`, it has nothing to do in `ExploreV2MenuComponentController`

Reply:
Fixed!